perf/bloom_filter/launch.lua.in

```diff
@@ -0,0 +1,105 @@
+#! @TARANTOOL_BINARY_DIR@/src/tarantool
+
+-- Benchmark is split into two sources: Lua (this one) and C. The reason is
+-- following: on the one hand C API does not provide convenient interface to
+-- create spaces and define its indexes and format. On the other hand Lua may
+-- spoil benchmark results e.g. due to the GC.
+
+local function greetings()
+    io.write("\n*****************************************************************\n")
+    io.write("\n*********          Bloom filter Perf Test            *******\n")
+    io.write("\n*****************************************************************\n")
+end
+
+
+local bench_run_func = "bench.run"
+local bench_init_func = "bench.init"
+local bench_stop_func = "bench.stop"
+local bench_funcs = { bench_run_func, bench_init_func, bench_stop_func }
+local primary_index_name = "pk"
+-- naming is followed in order to Lua pairs() call iterate through them correct
```

Comment:
I didn't get the idea...

Reply:
As soon as Lua `pairs` function does not iterate through `spaces_lookup_cost_coeffs_pairs` table values in the order they were declared, naming convention of view like "read<Number><Name>" at first seemed good for solving the order issue.
Removed comment with strange spaces naming.

Reply:
Update: decided to leave the naming as it's more comfortable to analyze such names while looking at benchmark results (they are logically sorted in lexicographic order).